src/Events/Admin/Onboarding/Landing_Page.php

```diff
@@ -207,25 +186,38 @@ public function admin_page_main_content(): void {
 	 */
 	public function admin_content_checklist_section(): void {
 		$settings_url   = 'edit.php?page=tec-events-settings&post_type=tribe_events';
+		$action_url     = add_query_arg( [ 'action' => self::DISMISS_PAGE_ACTION ], admin_url( '/admin-post.php' ) );
 		$data           = tribe( Data::class );
 		$completed_tabs = array_flip( (array) $data->get_wizard_setting( 'completed_tabs', [] ) );
 		$et_installed   = Installer::get()->is_installed( 'event-tickets' );
 		$et_activated   = Installer::get()->is_active( 'event-tickets' );
 		$organizer_data = $data->get_organizer_data();
 		$venue_data     = $data->get_venue_data();
 		$has_event      = $data->has_events();
+		$condition      = [
+			'1' => isset( $completed_tabs[1] ) || ! empty( tribe_get_option( 'tribeEnableViews' ) ),
+			'2' => isset( $completed_tabs[2] ) || ! empty( tribe_get_option( 'defaultCurrencyCode' ) ),
+			'3' => isset( $completed_tabs[3] ) || ! empty( tribe_get_option( 'dateWithYearFormat' ) ),
+			'4' => isset( $completed_tabs[4] ) || ! empty( $organizer_data ),
+			'5' => isset( $completed_tabs[5] ) || ! empty( $venue_data ),
+		];
+		$count_complete = count( array_filter( $condition ) );
 		?>
-			<div class="tec-admin-page__content-section tec-events-admin-page__content-section">
-				<h2 class="tec-admin-page__content-header"><?php esc_html_e( 'First-time setup', 'the-events-calendar' ); ?></h2>
+			<div class="tec-admin-page__content-section">
+				<div class="tec-admin-page__content-section-header">
+					<h2 class="tec-admin-page__content-header"><?php esc_html_e( 'First-time setup', 'the-events-calendar' ); ?></h2>
+					<a class="tec-dismiss-admin-page" href="<?php echo esc_url( $action_url ); ?>"><?php esc_html_e( 'Dismiss this screen', 'the-events-calendar' ); ?></a>
+				</div>
+				<div class="tec-admin-page__content-section-subheader"><?php echo esc_html( $count_complete ) . '/5 ' . esc_html__( 'steps completed', 'the-events-calendar' ); ?></div>
```

Comment:
If we do a count (all, not just complete) we can use that for the hardcoded "5" and not have to change it here if we add a tab 🤔